### ISSUE TITLE
Support request: error configuring network namespace [...] CNI network "bonsai_bonsai-net" not found

### ISSUE THREAD

**Issue by samuell** [Clinical-Genomics-Lund/bonsai#132]
I just thought to ask if you have any tips to share when getting the following errors, trying to do `docker-compose up -d` (after fixing an initial permissions issue with `sudo chown -R root:docker /run/podman/` and `sudo chmod g+rx /run/podman/`):

Running:
```bash
docker-compose up -d |& tee out.log
```
... I get:
```
Starting mongodb ... 
Starting bonsai_redis_1 ... 
Starting bonsai_redis_1 ... error

ERROR: for bonsai_redis_1  Cannot start service redis: error configuring network namespace for container 82379018a770d08c713825dca5cafecaf963451b5d0377c4ddad9c91ceebc2be: CNI network "bonsai_bonsai-net" not found
Starting mongodb        ... error

ERROR: for mongodb  Cannot start service mongodb: error configuring network namespace for container 1fc3741d41ae6296688ad8694c42a6d68de1fe63ef3e5c469bdcb213fe108cce: CNI network "bonsai_bonsai-net" not found

ERROR: for redis  Cannot start service redis: error configuring network namespace for container 82379018a770d08c713825dca5cafecaf963451b5d0377c4ddad9c91ceebc2be: CNI network "bonsai_bonsai-net" not found

ERROR: for mongodb  Cannot start service mongodb: error configuring network namespace for container 1fc3741d41ae6296688ad8694c42a6d68de1fe63ef3e5c469bdcb213fe108cce: CNI network "bonsai_bonsai-net" not found
Encountered errors while bringing up the project.
```

**Comment by mhkc** [comment]
Can you post your docker-compose file?

**Comment by samuell** [comment]
Got this running with the help of @mhkc now!

Turned out I had accidentally installed podman, which "hijacked" the docker command with an alias pointing to podman, and it turns out they are not 100% compatible. E.g. there were complaints that the CNI version used is not supported.

After installing the `docker.io` package (in Linux Mint 21.1), things worked much better. Also I did upgrade to 0.2.0 instead of 0.1.0 of Bonsai.
